collectors/python.d.plugin/mysql/mysql.chart.py

```diff
@@ -11,6 +11,7 @@
 QUERY_GLOBAL = 'SHOW GLOBAL STATUS;'
 QUERY_SLAVE = 'SHOW SLAVE STATUS;'
 QUERY_VARIABLES = 'SHOW GLOBAL VARIABLES LIKE \'max_connections\';'
+QUERY_USER_STATISTICS = 'SHOW USER_STATISTICS;'
```

Comment:
Do we need additional priviliges to make this query?

Reply:
users who have the SUPER or PROCESS will see the data in the table.

Reply:
this should be reflected in the module README

Reply:
added